### ISSUE TITLE
Error NT_XENT

### ISSUE THREAD

**Issue by olayaperezmon** [olayaperezmon/SimCLR#1]
Al ejecutar el script de SimCLR, en el apartado de Start Training, me da error al llamar a la loss function (nt_xent). El error se produce sólo en la ultima iteración. Creía que era porque quedaba un trozo más pequeño en la ultima iteracion que en los demas, pero aun quedando un número del mismo tamaño que en las demas iteraciones no solucioné el error.

**Comment by pglez82** [comment]
@olayaperezmon Necesito saber que simclr estás usando:

```from simclr import SimCLR```

¿Se supone que esto es una librería? ¿como la instalaste?

**Comment by olayaperezmon** [comment]
Sí, es una libreria. La instalé añadiendo en el notebook  `%pip install simCLR` 
y también en la máquina `python3 -m pip install simCLR`

**Comment by olayaperezmon** [comment]
Genial, muchísimas gracias!!
